packages/commerce-sdk-react/src/hooks/ShopperExperience/query.ts

```diff
@@ -52,9 +54,13 @@ export const usePages = (
     })
 }
 /**
- * Get a Page Designer page based on a single page ID. The results will apply the visibility rules for the page's components, such as personalization or scheduled visibility.
-
-**Important**: Because this resource uses the GET method, you must not pass sensitive data (payment card information, for example) and must not perform any transactional processes within the server-side scripts that are run for the page and components.
+ * Get a Page Designer page based on a single page ID.
+ *
+ * @remarks
+ * The results will apply the visibility rules for the page's components, such as personalization or scheduled visibility.
+ * *Important**: Because this resource uses the GET method, you must not pass sensitive data (payment card information, for example) and must not perform any transactional processes within the server-side scripts that are run for the page and components.
```

Comment:
```suggestion
 **Important**: Because this resource uses the GET method, you must not pass sensitive data (payment card information, for example) and must not perform any transactional processes within the server-side scripts that are run for the page and components.
```